frontend/src/components/Worksheet/WorksheetStats.tsx

```diff
@@ -39,18 +42,32 @@ export default function WorksheetStats() {
     rating,
     skillsAreas,
   } = courses.reduce(
-    (acc, c) =>
-      hidden_courses[cur_season]?.[c.crn]
-        ? acc
-        : {
-            courseCnt: acc.courseCnt + 1,
-            coursesWithRating:
-              acc.coursesWithRating + (c.average_rating ? 1 : 0),
-            credits: acc.credits + (c.credits ?? 0),
-            workload: acc.workload + (c.average_workload ?? 0),
-            rating: acc.rating + (c.average_rating ?? 0),
-            skillsAreas: [...acc.skillsAreas, ...c.skills, ...c.areas],
-          },
+    (acc, c) => {
+      //const sectionIsInt = Number.isInteger(parseInt(c.section));
+      // see if any of the course's codes have already been counted or if it's hidden so we don't double count
+      const shouldNotCount =
+        c.all_course_codes.some((code) => countedCourseCodes.has(code)) ||
+        hidden_courses[cur_season]?.[c.crn];
+      const hasCourseInfo = c.average_rating && c.credits && c.average_workload;
```

Comment:
If a course has no rating but has credits, we should count it nontheless, no?

Reply:
Ok so the scenario is just when it has no credits skip, otherwise treat it as the one to get data from?

Reply:
Yeah that sounds reasonable